mods/sp/weapons/cabweapons.yaml

```diff
@@ -412,6 +430,7 @@ DroneHostShellSpawner:
 	Warhead@1Dam: SpreadDamage
 		Spread: 1
 		Damage: 1
+	-Warhead@2Eff:
```

Comment:
I would keep the explosion of the rocket since the it has to split to deploy the rest.

Reply:
I tried but the explosion animation cover the make animation of drone